tests/test_wcs.py

```diff
@@ -408,26 +408,43 @@ def test_ndvi_xarray(self):
                     [255, 255, 115, 255, 139, 255, 255, 255],
                     [255, 255, 255, 255, 255, 255, 255, 255],
                     [255, 255, 255, 255, 255, 255, 255, 255]
-                ]]),
+                ]], dtype=np.uint8),
                 coords={
                     'band': [1],
+                    'x': [-157.5, -112.5, -67.5, -22.5, 22.5, 67.5, 112.5, 157.5],
                     'y': [78.75, 56.25, 33.75, 11.25, -11.25, -33.75, -56.25, -78.75],
-                    'x': [-157.5, -112.5, -67.5, -22.5, 22.5, 67.5, 112.5, 157.5]
+                    'spatial_ref': 0,
                 },
                 dims=["band", "y", "x"],
                 attrs={
-                    'transform': (45.0, 0.0, -180.0, 0.0, -22.5, 90.0),
-                    'crs': '+init=epsg:4326',
-                    'res': (45.0, 22.5),
-                    'is_tiled': False,
-                    'nodatavals': (0.0,),
-                    'scales': (1.0,),
-                    'offsets': (0.0,),
-                    'AREA_OR_POINT': 'Area',
+                    'transform': (45.0, 0.0, -180.0, 0.0, -22.5, 90.0, 0.0, 0.0, 1.0),
+                    'crs': 'EPSG:4326',
+                    'res': (45.0, -22.5),
+                    'scale_factor': 1.0,
+                    '_FillValue': 0.0,
+                    'add_offset': 0.0,
                 },
             )
 
-            self.assertTrue(array.identical(expected), msg=f'{array}\n!=\n{expected}')
+            # https://github.com/corteva/rioxarray/blob/ca62a67a3db3aa62afc85e57bb6153dbc503e1dd/test/integration/test_integration_rioxarray.py
```

Comment:
```suggestion
```

I'm not quite sure why the URL is there.

Reply:
Removed in [f9b3e18](https://github.com/geo-engine/geoengine-python/pull/72/commits/f9b3e1882942a582f617884cebf2ee8a61c4c791)